lib/control-app.js

```diff
@@ -25,10 +41,15 @@ module.exports = function(config) {
 
     var build;
     // default builder is browserify which we provide
-    config.builder = config.builder || '../lib/builder-browserify';
+    config.builder = config.builder || defaultBuilder;
```

Comment:
> Looks great otherwise! :)

Thanks, I am using this fork on my projects since some days, you can tell the performance difference when using --local